### PR TITLE
add: `signal-desktop-deb`

### PR DESCRIPTION
**Pull request by jonafato**
- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

File: packages/signal-desktop-deb/signal-desktop-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name="signal-desktop-deb"
+pkgname="signal-desktop"
+gives="signal-desktop"
+replace="signal-desktop"
```
Comment (Henryws) [inline packages/signal-desktop-deb/signal-desktop-deb.pacscript:4]:
```suggestion
gives="${pkgname}"
replace="${pkgname}"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/signal-desktop-deb/signal-desktop-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name="signal-desktop-deb"
+pkgname="signal-desktop"
+gives="${pkgname}"
+replace="${pkgname}"
+repology=("project: signal-desktop")
```
Comment (WRM-42) [inline packages/signal-desktop-deb/signal-desktop-deb.pacscript:5]:
```suggestion
repology=("project: ${pkgname}")
```

**Review by WRM-42 — CHANGES_REQUESTED** [review]
Add `breaks`

**Comment by jonafato** [comment]
@WRM-42 Thanks for the review.

> Add `breaks`

Should this also be `${pkgname}`? This is my first contribution here, so I'm still learning the project conventions.

**Comment by WRM-42** [comment]
> Should this also be `${pkgname}`? This is my first contribution here, so I'm still learning the project conventions.

You should also add `${pkgname}`-beta and `${pkgname}`-app, since Signal has these two versions

**Comment by jonafato** [comment]
> > Should this also be `${pkgname}`? This is my first contribution here, so I'm still learning the project conventions.
> 
> You should also add `${pkgname}`-beta and `${pkgname}`-app, since Signal has these two versions

Per the note copied below from [the beta support article](https://support.signal.org/hc/en-us/articles/360007318471-Signal-Beta), the beta is a separate install from the regular desktop app and explicitly does not conflict.

>This is a new install with no message history. It is separate from the [production version of Signal Desktop](https://signal.org/download) and will not replace it.

I'm happy to add the `-app` version here, though I'm not sure I see where Signal provides an appimage build (or am I misunderstanding the meaning of `-app` in this context?).

(I'm planning on packaging the beta as well once this addition is in a good merge-ready condition.)

**Comment by Henryws** [comment]
What `${pkgname}-app` means, when put in `breaks` is that *if* an appimage was released, this pacsctipt would be able to detect that.

**Comment by WRM-42** [comment]
>Per the note copied below from [the beta support article](https://support.signal.org/hc/en-us/articles/360007318471-Signal-Beta), the beta is a separate install from the regular desktop app and explicitly does not conflict.

Thanks for mentioning this

> I'm happy to add the `-app` version here, though I'm not sure I see where Signal provides an appimage build (or am I misunderstanding the meaning of `-app` in this context?).

I've seen some people using AppImages of Signal, but they're not official

**Review by WRM-42 — APPROVED** [review]
++

**Pull request merged**